<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>securiteStudio</class>
 <widget class="QDialog" name="securiteStudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tv'Ease</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/img/images/logo.png</normaloff>:/img/images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #9AC8CD ;</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-top : 100px ;
padding-bottom : 100px ;
color : #0E46A3 ;</string>
   </property>
   <property name="text">
    <string>Fiche Accees personnels - studios</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton__securite">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent ; </string>
   </property>
   <property name="text">
    <string>Telechargement</string>
   </property>
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/img/images/pd.png</normaloff>:/img/images/pd.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Papyrus</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color :#333770 ;</string>
   </property>
   <property name="text">
    <string>Afiichage du personnels  -  studios </string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_securite">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>981</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color:  white; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_recher_securite">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/img/images/search.png</normaloff>:/img/images/search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_recher_securite">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : white ; 
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_trie_securite">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>140</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : white ; </string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/img/images/table.png</normaloff>:/img/images/table.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>trie dans l'ordre decroissant par date acces</string>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/img/images/decroissant.png</normaloff>:/img/images/decroissant.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>trie dans l'ordre croissant par date acces</string>
    </property>
    <property name="icon">
     <iconset resource="source.qrc">
      <normaloff>:/img/images/croissant.png</normaloff>:/img/images/croissant.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trier ...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
